manifests/base/templates/endpoints.yaml

```diff
@@ -1,3 +1,37 @@
+{{- if .Values.global.remotePilotAddress }}
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: istio-pilot
+  namespace: {{ .Values.global.istioNamespace }}
+subsets:
+- addresses:
+  - ip: {{ .Values.global.remotePilotAddress }}
+  ports:
+  - port: 15010
+    name: grpc-xds # direct
+  - port: 15011
+    name: https-xds # mTLS or non-mTLS depending on auth setting
+  - port: 8080
```

Comment:
This may be a problem port - can you test without in a pre istiod/post istiod world? (1.4 + 1.5 two cluster setup?)